manifests/settings_file.pp

```diff
@@ -33,7 +33,13 @@
   if $ensure == 'absent' {
     $content = undef
   } else {
-    $content = template($template_path)
+    $content = if $template_path.match(/\.epp$/) {
+      epp($template_path)
```

Comment:
It'd be nice if we could pass in an explicit context as well.

Reply:
What exactly do you mean by "context"?

Reply:
You can call `epp($template_path, $context)` so I'm thinking about a new parameter to the defined type.

Reply:
That`s fine by me, but in a different Pullrequest then.